Core/include/Acts/Seeding/SeedFilter.ipp

```diff
@@ -189,17 +190,12 @@ void SeedFilter<external_spacepoint_t>::filterSeeds_2SpFixed(
       if (deltaSeedConf > 0) {
         // if we have not yet reached our max number of quality seeds we add the
         // new seed to outCont
+
+        candidates_collector.push(bottomSP, middleSP, *topSpVec[topSPIndex],
+                                  weight, zOrigin, true);
```

Comment:
Can you extend the comment mentioning that `push` also checks the max number of quality seeds for a middle sp? If this is reached, we remove a seed with lower weight.